### ISSUE TITLE
[BUG] Mystery "Could not get Repo Listing" error in VCC (but not VPM CLI)

### ISSUE THREAD

**Issue by bdunderscore** [vrchat-community/creator-companion#5]
**Describe the bug**
The VCC fails to load the custom repository https://vpm.fushizen.net/vpm-prerelease.json with a very unhelpful "[WRN] Could not get Repo Listing from [url]" message.

**To Reproduce**
1. Add the repo using the VPM CLI: `vpm add repo https://vpm.fushizen.net/vpm-prerelease.json`
2. Open the VCC
3. The "Could not get Repo Listing" error will appear in the bottom of the VCC screen, and the repo will not be usable.

**Expected behavior**
The VCC should be able to use this repo, because the VPM CLI is able to use it:
```
λ vpm list repo https://vpm.fushizen.net/vpm-prerelease.json
[18:46:55 INF]
[18:46:55 INF] Modular Avatar | net.fushizen.modular-avatar
[18:46:55 INF]
[18:46:55 INF] 0.1.1: https://github.com/bdunderscore/modular-avatar/releases/download/0.1.1/net.fushizen.modular-avatar-0.1.1.zip
[18:46:55 INF] 0.1.0: https://github.com/bdunderscore/modular-avatar/releases/download/0.1.0/net.fushizen.modular-avatar-0.1.0.zip
```

**Screenshots**
![image](https://user-images.githubusercontent.com/64174065/189464280-2f1917e3-9ac7-4180-ac17-bde1726a821d.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - VCC version: 1.0.0

**Additional context**
Appending a `?` to the URL makes this work. I guess that fix hasn't made it to the VCC GUI version?

**Comment by momo-the-monster** [comment]
@bdunderscore - you are correct that the VPM CLI has been updated with the fix that required ? in the URL, and the VCC has not (it is available in the VCC beta of 1.0.1, but you'll need to copy the `userRepo` listing from the settings.json file in release into the one in Beta).

**Comment by momo-the-monster** [comment]
Fixed in 1.0.1 by changing the way dependency versions are checked when building the dependency tree.
